client/src/App.css

```diff
@@ -29,16 +28,27 @@
   color: white;
 }
 
-.halogen-line{
-height: 3px;
-
-
+.halogen-line {
+  height: 3px;
 }
 
 .App-link {
   color: #1f657a;
 }
 
+.lightbox-sub {
+  width: 95%;
+  height: 95%;
+}
+
+.lightbox-circle {
+  border: 90px solid #fff;
+  height: 705px;
```

Comment:
We might want to consider changing these to percentage values so that it looks good on different screen sizes as well